src/app/(legal)/layout.tsx

```diff
@@ -0,0 +1,9 @@
+import '@/app/globals.css'
```

Comment:
You never imported `React`. Add `import React, { type ReactNode } from 'react'` to the top, and use `ReactNode` directly on line 6.